### PR TITLE
Feature/#153 webgl minimal

### PR DESCRIPTION
**Pull request by leon-juenemann**
My second attempt to draw the channels with WebGL.
Now with fewer changes to the existing code structure. Was simply not feasible otherwise to merge after all the new changes. 

I tried out a lot of different ways to draw the signals. In the end, I opted for this easy approach with minimal changes to the code. 

Signal heads are missing. -> new ticket.

### CODE REVIEW

**Comment by leon-juenemann** [comment]
Performance is a lot better since last commit (webgl.DYNAMIC_DRAW)

**Review by motschel123 — APPROVED** [review]
For me it doesn't seem like STATIC_DRAW vs DYNAMIC_DRAW makes much of a difference, but both implementation quite well on my machine. Great work

File: Apps/frontend/src/components/Waves.svelte [hunk 1]
```diff
@@ -1,51 +1,39 @@
 <script>
   import { beforeUpdate, onMount } from "svelte";
-  import {
-    ColorRGBA,
-    WebglPlot,
-    WebglSquare,
-    WebglThickLine,
-  } from "webgl-plot";
+  import App from "../App.svelte";
```
Comment (PhlppKrmr) [inline Apps/frontend/src/components/Waves.svelte:3]: Unused import.

File: Apps/frontend/src/components/Waves.svelte [hunk 2]
```diff
@@ -100,94 +88,34 @@
...
-  });
-
-  // Subscribe to the channelActivation store
-  channelActivated.subscribe((isActive) => {
-    for (let i = 0; i < NUM_CHANNELS; i++) {
-      if (lines[i] !== undefined) lines[i].visible = isActive[i];
-    }
-  });
-
   thicknessAdjustment.subscribe((isThick) => {
```
Comment (PhlppKrmr) [inline Apps/frontend/src/components/Waves.svelte:91]: `isThick` unused.

**Review by PhlppKrmr — APPROVED** [review]
Looks very good as a base we can work with. Only have a few cleanup recommendations.

Apart from that I again tested the (time sweep) slider performance loss bug:
If we change the time sweep a lot we eventually run into a performance loss. This happens both in your and in the webgl-plot implementation, in yours it is merely more pronounced (earlier noticeable).

In the beginning:
image

After chaning the time sweep a few times:
image

I personally would still say we can merge this current version and build upon it with (at least) one person taking on the task of improving performance.

**Review by nicolaskolbenschlag — APPROVED** [review]
Works great. Good work!

File: Apps/frontend/src/components/Waves.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (leon-juenemann) [reply Apps/frontend/src/components/Waves.svelte]: done

File: Apps/frontend/src/components/Waves.svelte [hunk 3]
```diff
@@ -1,51 +1,39 @@
 <script>
```
[the remark this answers is not in the document]
Comment (leon-juenemann) [reply Apps/frontend/src/components/Waves.svelte:1]: done. but did not change anything.

**Review by PhlppKrmr — APPROVED** [review]

**Pull request merged**